src/components/Button/Button.test.jsx

```diff
@@ -142,9 +143,9 @@ describe('Button', () => {
         expect(buttonElement).toBeInTheDocument();
       });
 
-      test('it renders an anchor tag if an href attribute is passed as a prop', () => {
+      test('it renders an anchor tag if as prop `a` is passed', () => {
```

Comment:
this test is passing, but Chromatic says differently: https://www.chromatic.com/snapshot?appId=5ed9214b642dc10022b50a2d&id=5f9080a3409e4d0022907fcb

Reply:
@nathanyoung I neglected to update the chromatic story, but the test has the correct implementation. Story is updating now.